src/main/java/net/dv8tion/jda/internal/managers/ChannelManagerImpl.java

```diff
@@ -171,8 +173,23 @@ public ChannelManagerImpl putPermissionOverride(@Nonnull IPermissionHolder permH
     {
         Checks.notNull(permHolder, "PermissionHolder");
         Checks.check(permHolder.getGuild().equals(getGuild()), "PermissionHolder is not from the same Guild!");
-        if (isPermissionChecksEnabled() && !getGuild().getSelfMember().hasPermission(getChannel(), Permission.MANAGE_PERMISSIONS))
-            throw new InsufficientPermissionException(getChannel(), Permission.MANAGE_PERMISSIONS);
+        Member selfMember = getGuild().getSelfMember();
+        if (isPermissionChecksEnabled() && !selfMember.hasPermission(Permission.ADMINISTRATOR))
```

Comment:
You also can only set `MANAGE_PERMISSIONS` on a channel if you have it as an override on the channel or `ADMINISTRATOR` on a role.